frame/multisig/src/tests.rs

```diff
@@ -328,63 +234,19 @@ fn timepoint_checking_works() {
 				2,
 				vec![1, 3],
 				None,
-				OpaqueCall::from_encoded(call.clone()),
-				false,
+				call.clone(),
 				Weight::zero()
 			),
 			Error::<Test>::NoTimepoint,
 		);
 		let later = Timepoint { index: 1, ..now() };
 		assert_noop!(
-			Multisig::as_multi(
-				Origin::signed(2),
-				2,
-				vec![1, 3],
-				Some(later),
-				OpaqueCall::from_encoded(call),
-				false,
-				Weight::zero()
-			),
+			Multisig::as_multi(Origin::signed(2), 2, vec![1, 3], Some(later), call, Weight::zero()),
 			Error::<Test>::WrongTimepoint,
 		);
 	});
 }
 
-#[test]
-fn multisig_2_of_3_works_with_call_storing() {
```

Comment:
similarly, should this test scenario remain, just without call storing?

Reply:
AFAIK this already exists in another test.

Reply:
duplicate test https://github.com/paritytech/substrate/blob/7bacd730ff42afe9d6c5039a51e74bd461dac5ef/frame/multisig/src/tests.rs#L251